### DISCUSSION TITLE
I can't trigger events with Livewire

### DISCUSSION THREAD

**Discussion by gotzsys** [filamentphp/filament#8169] [Help] [answered]
```
protected function getStats(): array
{
    return [
        Stat::make('Processed', '192.1k')
            ->color('success')
            ->extraAttributes([
                'class' => 'cursor-pointer',
                'wire:click' => '$dispatch("setStatusFilter", "processed")',
            ]),
        // ...
    ];
}
```
Doing it this way, it's escaping the characters
![image](https://github.com/filamentphp/filament/assets/13874850/433e2970-7eda-42c5-985a-d8bde21fb4ad)

![image](https://github.com/filamentphp/filament/assets/13874850/915d1af6-2a67-458a-9bb9-632ea037ec6f)

I tried other ways but it didn't work.
'wire:click' => '$dispatch("setStatusFilter", { status: "waiting" })',
"wire:click" => "\$dispatch('setStatusFilter', { status: 'waiting' })",

**Comment by danharrin** [comment]
So what does using `'` produce

**Comment by leandrocfe** [comment, marked as answer]
```php
->extraAttributes(['wire:click' => '$dispatch(\'setStatusFilter\', { status: \'processed\' })'])
```
